aicssegmentation/workflow/workflow_engine.py

```diff
@@ -42,6 +44,26 @@ def get_executable_workflow(self, workflow_name: str, input_image: np.ndarray) -
 
         return Workflow(definition, input_image)
 
+    def _load_workflow_def(self, file_path: path) -> WorkflowDefinition:
+        if not path.exists(file_path):
+            raise FileNotFoundError(f"Did not find a file at {file_path}")
+        if path.splitext(file_path)[1].lower() != ".json":
+            raise FileNotFoundError(f"The file at {file_path} is not a json file.")
```

Comment:
The file exists here so don't throw a `FileNotFoundError`. Use some other exception, maybe `ValueError`.